mixer/adapter/list/list_test.go

```diff
@@ -503,7 +503,8 @@ func TestRefreshAndPurge(t *testing.T) {
 	// wait for the list to have been populated
 	for {
 		time.Sleep(1 * time.Millisecond)
-		if h.hasData() {
+		result, _ := h.hasData()
+		if result {
```

Comment:
do we care about timing out at some point?  or are we fine with the default timeout on tests?

Reply:
Do you think it's needed for reliability of the test? I thought the timescales involved should work out well (refresh is in a few milliseconds, test timeout is a few seconds)